Examples/testCustomEntitlementsComputation/testCustomEntitlementsComputation/ContentView.swift

```diff
@@ -0,0 +1,278 @@
+    @Binding var inputText: String
+
+    let logIn: (String) -> Void
+
+    var body: some View {
+        VStack(spacing: 20) {
+            Text("Log In with different user:")
+                .font(.headline)
+
+            TextField("Enter App User ID here", text: $inputText)
+                .textFieldStyle(RoundedBorderTextFieldStyle())
+                .padding(.horizontal)
+
+            HStack {
+                Button("Cancel") {
+                    dismiss()
+                }
+                .padding()
+                .background(Color.red)
+                .cornerRadius(10)
+                .foregroundColor(.white)
+
+                Button("Log In") {
+                    logIn(inputText)
+                    dismiss()
+                }
+                .padding()
+                .background(Color.green)
+                .cornerRadius(10)
+                .foregroundColor(.white)
+            }
+        }
+        .padding()
+    }
+}
+
+struct ExplanationView: View {
+    var body: some View {
+        VStack {
+            Text("Custom Entitlements Mode")
+                .font(.largeTitle)
+                .padding()
+
+            Text("This mode is intended for apps that will do their own entitlement computation, " +
+                 "separate from RevenueCat. ")
+            .padding()
+            Text("In this mode, RevenueCat will not generate " +
+                 "anonymous user IDs, it will not refresh customerInfo cache automatically " +
+                 "(only when a purchase goes through), and it will disallow the logOut methods.")
+            .padding()
+            Text("Use logIn to switch to a different App User ID if needed. The SDK should only be configured once" +
+                 "the initial appUserID is known.")
+            .padding()
+            Text("Apps using this mode rely on webhooks to signal their backends to refresh " +
+                 "entitlements with RevenueCat.")
+            .padding()
+
+            Spacer()
+        }
+        .padding()
+    }
+}
+
+struct CustomerInfoDetailsView: View {
+    let customerInfo: CustomerInfo
+
+    var prettyJSON: String {
+        let encoder = JSONEncoder()
+        encoder.outputFormatting = .prettyPrinted
+
+        if let jsonData = try? encoder.encode(customerInfo),
+           let jsonString = String(data: jsonData, encoding: .utf8) {
+            return jsonString
+        } else {
+            return "Error: Unable to convert CustomerInfo to JSON."
+        }
+    }
+
+    var body: some View {
+        VStack(alignment: .leading) {
+            ScrollView {
+                Text(prettyJSON)
+                    .padding()
+                    .font(.system(.body, design: .monospaced))
+            }
+
+            Spacer()
+        }
+        .padding()
+        .navigationTitle("Customer Info Details")
+    }
+}
+
+struct ContentView_Previews: PreviewProvider {
+    static var previews: some View {
+        ContentView()
+    }
+}
```

Comment:
This file has quite a few things... But I think it's ok for a quick sample app.

Reply:
I genuinely debated whether to split this up when writing it, since adding more files in swift is pretty trivial. Not entirely sure it's a net win for this - it feels like if you're using this as a reference, it's kinda nice for all the logic to be in one place so you don't need to figure out how to navigate it. Especially given that it's not that long of a file

Reply:
thoughts?